src/pandroid/app/src/main/java/com/panda3ds/pandroid/view/PandaGlRenderer.java

```diff
@@ -61,13 +67,24 @@ public void onSurfaceCreated(GL10 unused, EGLConfig config) {
         AlberDriver.Initialize();
     }
 
+    public void postDrawEvent(Runnable callback){
+        events.add(callback);
```

Comment:
What is this for? Is it for running code on the gl thread? Can't we just ues GlSurfaceView.queueEvent for that?

Reply:
*queueEvent* can used, but caused crash on my device, this can replaced by Handler.postDelayed or can replace from the String with ROM path from the constructor

Reply:
queueEvent crashed because the glrenderer didn't have enough time to initialize